bigtable-dataflow-parent/bigtable-hbase-beam/src/main/java/com/google/cloud/bigtable/beam/CloudBigtableScanConfiguration.java

```diff
@@ -103,6 +103,54 @@ Builder withRequest(ValueProvider<ReadRowsRequest> request) {
       return this;
     }
 
+    /**
+     * Provides an updated request by adding a row range with start and end keys in the existing
+     * request.
+     */
+    private static class RequestWithKeysValueProvider
+        implements ValueProvider<ReadRowsRequest>, Serializable {
+      private final ByteString start;
+      private final ByteString stop;
+      private final ValueProvider<ReadRowsRequest> request;
+      private transient volatile ReadRowsRequest cachedRequest;
```

Comment:
nit: I dont think you need transient (ReadRowsRequest is serializable) and I don't think you need volatile because this is is not threadsafe

Reply:
done.